daemon/api_quotas.go

```diff
@@ -63,8 +63,22 @@ var (
 	servicestateRemoveQuota = servicestate.RemoveQuota
 )
 
-var getQuotaMemUsage = func(grp *quota.Group) (quantity.Size, error) {
-	return grp.CurrentMemoryUsage()
+var getQuotaUsage = func(grp *quota.Group) (*client.QuotaValues, error) {
+	var currentUsage client.QuotaValues
```

Comment:
No strong opinion, but I'm wondering if you should use the same struct (QuotaValues) for both the constrains and current quota usage. Maybe others have opinion?

Reply:
I don't really have a strong opinion on this either, if anyone feels strongly about changing this I will, otherwise I'll leave the implementation like this

Reply:
Good point about daemon vs client, but my point was about reusing same struct for both setting the limit and obtaining current usage. But maybe I'm overthinking this.

Reply:
I think we should leave the implementation like this as we do not have any requirements or need for this to use two different structs. Lets change it in the future if we see it neccessary. I'll resolve this unless anyone have strong objections.